src/symbol.rs

```diff
@@ -1,6 +1,7 @@
 //! This module provides a naive implementation of symbolication for the time being.
 //! It should be expanded to support multiple data sources.
 
+#[macro_use]
```

Comment:
I think we can follow the 2018 edition style here, so we can replace this with
```rust
use dwarf_utils::dwarf_attr;
```

Reply:
As far as I know that only works for extern crates and the unstable decl macro 2.0.